upgrade_tests/helpers/existence.py

```diff
@@ -375,17 +367,17 @@ def compare_postupgrade(component, attribute):
     predata = get_datastore('preupgrade', endpoint)
     postdata = get_datastore('postupgrade', endpoint)
     entity_values = []
-    atr = 'id' if endpoint == 'api' else cli_const.attribute_keys[component]
-    for test_case in find_datastore(predata, component, atr):
+    atr = 'id' if endpoint == 'api' else CLI_ATTRIBUTES_KEY[component]
+    for _ in find_datastore(predata, component, atr):
```

Comment:
Any reason why `test_case` replaced with `_` ?

Reply:
The intention was to reduce the unnecessary variable uses, but as per the offline discussion the variable name will help to understand the iterating item type.